addons/calendar/views/calendar_views.xml

```diff
@@ -72,7 +72,7 @@
         <field name="name">calendar.event.list</field>
         <field name="model">calendar.event</field>
         <field name="arch" type="xml">
-            <list string="Meetings" sample="1" multi_edit="1" js_class="calendar_list_view">
+            <list string="Meetings" sample="1" multi_edit="1" js_class="calendar_list_view" delete="0">
```

Comment:
It could be super painful to delete event one by one in list view if we have to delete them. Having to create a server action for that is a bit annoying. Is this really what we want?

Reply:
The `delete="0"` attribute here disables the generic delete functionality. We made server action for delete instead because we can not add some flow in generic delete As we want to send an email to the users on event deletion with our change also we can delete multiple events at a time not one by one. This is how this flow has been handled in the codebase too.

Reply:
@arj-odoo , If I understand  correctly your concern
Yes, we need to remove the action from the list view. I suggest we do this after completing this [task](https://www.odoo.com/odoo/project/4107/tasks/4063288), as it includes improvements for deleting recurring events from the list view.

Recurrence is not supported in the list view, so deleting one event does not delete other related recurring events. I recommend waiting until the above task is merged, as it harmonizes the deletion flow for recurring events across form, list, and calendar views. Remove changes related to the list view from this PR.
What do you think?

Or do you want to say something else? 🤔